src/node.ts

```diff
@@ -97,10 +102,21 @@ export class EthereumNode extends constructs.Construct {
     // will be populated with defaults when passed to the node constructor
     this.network = props.network ?? Network.MAINNET;
     this.instanceType = props.instanceType ?? InstanceType.BURSTABLE3_LARGE;
-    this.availabilityZone = props.availabilityZone ?? `${region}a`;
 
-    utilities.validateRegion(region);
-    utilities.validateAvailabilityZone(region, this.availabilityZone);
+
+    if (tokenRegion) {
+      // If the region is not set as an environment variable and the availability zone is not included in node instantiation,
```

Comment:
Add this to the list of properties, might as well have region as well as AZ.

  /**
     * The Region in which the node will be created
     * @default - us-east-1
     */
  readonly region?: string;

Reply:
I tried adding `region` as a node property, but seeing it does not populate `managedblockchain.cfnNode` at construction and the construct does not validate beyond the value assigned `region = cdk.Stack.of(this).region`, I do not think adding it to the props provides any value for the end user. Let me know what you think!

Reply:
I wasn't suggesting it as an _input_, I was suggesting it as an _output_, only to go along with availability zone.